deploy/helpers.ts

```diff
@@ -9,6 +9,8 @@ const exec = util.promisify(childProcess.exec);
 export const getArbitrumContracts = (network: string) => {
   if (['rinkeby', 'rinkebyDevnet', 'arbitrumRinkeby', 'arbitrumRinkebyDevnet'].indexOf(network) > -1) {
     return ARBITRUM_NETWORK.rinkeby;
+  } else if (['nitroDevnet', 'arbitrumNitroDevnet'].indexOf(network) > -1) {
```

Comment:
What's the difference between `arbitrumNitroDevnet` and `arbitrumRinkeby`? Shouldn't the contract addresses be the same between those, and not between the Goerli and Arbitrum based devnets?

Reply:
Arbitrum Nitro Devnet is actually Goerli based devnet only. The L1 (nitroDevnet) is the Goerli chain and the L2 (arbitrumNitroDevnet) listens to the txs on Goerli. I believe the confusion is maybe because of the naming convention I followed? It can also be called arbitrumGoerliDevnet, I just felt arbitrumNitroDevnet sounded a bit a better.
Did I understand the question correct?

Reply:
Ahh I see - yeah it was the naming convention that was confusing. Goerli isn't necessarily nitro, so it might be less confusing to rename. Like `goerli` and `goerliNitro` might be more straightforward, just my 2c!

Reply:
I see your point now, thanks! Considering that rinkeby will also be upgraded nitro, it will be nice to have an identifier for the network. Addressed this in 09d0156.